src/overlays/actors/ovl_Bg_Mizu_Water/z_bg_mizu_water.c

```diff
@@ -9,7 +15,23 @@ void BgMizuWater_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgMizuWater_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgMizuWater_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void BgMizuWater_Idle(BgMizuWater* this, GlobalContext* globalCtx);
+void BgMizuWater_ChangeWaterLevel(BgMizuWater* this, GlobalContext* globalCtx);
+
+extern UNK_TYPE D_06004B20;
+
+typedef struct {
+    s32 switchID;
+    s32 yDiff;
+} WaterLevel;
```

Comment:
Usually a switch flag is called `switchFlag`, may want to change `switchID` to that to keep it consistent.